pages/api/contact.js

```diff
@@ -0,0 +1,31 @@
+export default function (req, res) {
+  const nodemailer = require('nodemailer')
+  const transporter = nodemailer.createTransport({
+    // This is for mailtrap tester. Need to update to real SMTP settings, such as Gmail.
+    host: 'smtp.mailtrap.io',
+    port: 2525,
+    auth: {
+      user: process.env.EMAIL_USERNAME,
+      pass: process.env.EMAIL_PASSWORD
+    }
+  })
+
+  const mailData = {
+    // Customize to your SMTP requirements
+    from: process.env.EMAIL_USERNAME,
+    to: 'youremail@email.com', // Can be an array of recipients
```

Comment:
@joelwass where should contact email go? I'm assuming `joel@theteacherfund.com` ?

Reply:
Added process.env.CONTACT_EMAIL for this.

Reply:
even better so my email isn't scraped :)